lib/modules/datasource/packagist/schema.ts

```diff
@@ -69,6 +71,12 @@ export function parsePackagesResponses(
         dep.releaseTimestamp = composerRelease.time;
       }
 
+      phpConstraint ??= composerRelease.require?.php;
```

Comment:
```suggestion
      phpConstraint ??= composerRelease.require;
```

Reply:
Since we care for just single field inside nested structure, we transform it up for simpler retrieval.